Xamarin.Forms.Core/CornerRadius.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core
+{
+	public struct CornerRadius
+	{
+		public double TopLeft { get; set; }
```

Comment:
make it immutable by decorating the setter with `private`

Reply:
Done.